doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -0,0 +1,599 @@
+        make \
+        gnu-efi \
+        libssl-dev \
+        libpciaccess-dev \
+        uuid-dev \
+        libsystemd-dev \
+        libevent-dev \
+        libxml2-dev \
+        libusb-1.0-0-dev \
+        python3 \
+        python3-pip \
+        libblkid-dev \
+        e2fslibs-dev \
+        pkg-config \
+        zlib1g-dev \
+        libnuma-dev \
+
+        liblz4-tool
+
+      $ pip3 install kconfiglib
+
+#. Get the ACRN source code:
+
+   .. code-block:: none
+
+      $ cd /home/intel/work
+      $ git clone https://github.com/projectacrn/acrn-hypervisor
+      $ cd acrn-hypvervisor
+
+#. Switch to the v2.0 version:
+
+   .. code-block:: none
+
+      $ git checkout -b v2.0 remotes/origin/release_2.0
+
+4. Apply CAT and other patches if necessary [optional].
+
+5. Build ACRN:
+
+   .. code-block:: none
+
+      $ make all BOARD_FILE=misc/acrn-config/xmls/board-xmls/whl-ipc-i7.xml SCENARIO_FILE=misc/acrn-config/xmls/config-xmls/whl-ipc-i7/industry.xml RELEASE=0
+
+      $ make install
+
+Enable network sharing for the User VM
+======================================
+
+.. code-block:: none
+
+   $ sudo systemctl enable systemd-networkd
+   $ sudo systemctl start systemd-networkd
+
+Build and install the ACRN kernel
+=================================
+
+#. Build the Service VM kernel from opensource:
+
+   .. code-block:: none
+
+      $ cd /home/acrn/work/
+      $ git clone https://github.com/projectacrn/acrn-kernel
+
+#. Switch to the 5.4 kernel:
+
+   .. code-block:: none
+
+      $ git checkcout -b v2.0 remotes/origin/release_2.0
+      $ cp kernel_config_uefi_sos .config
+      $ make oldconfig
+      $ make all
+      $ sudo make modules_install
+
+Install the Service VM kernel and modules
+=========================================
+
+.. code-block:: none
+
+   $ sudo cp -r ~/sos-kernel-build/usr/lib/modules/4.19.78-98.iot-lts2018-sos/ /lib/modules
+   $ sudo mkdir /boot/acrn/
+   $ sudo cp ~/sos-kernel-build/usr/lib/kernel/lts2018-sos.4.19.78-98  /boot/bzImage
+
+Copy the Service VM kernel files located at ``arch/x86/boot/bzImage`` to the ``/boot/`` folder.
+
+Update Grub for the Ubuntu Service VM
+=====================================
+
+#. Update the ``/etc/grub.d/40_custom`` file as shown below. 
+
+   .. note::
+      Enter the command line for the kernel in ``/etc/grub.d/40_custom`` as
+      a single line and not as multiple lines. Otherwise, the kernel will
+      fail to boot.
+
+   **menuentry 'ACRN Multiboot Ubuntu SOS' --id ubuntu-service-os**
+
+   .. code-block:: bash
+
+      {
+
+        load_video
+        insmod gzio
+        insmod part_gpt
+        insmod ext2
+
+        search --no-floppy --fs-uuid --set 9bd58889-add7-410c-bdb7-1fbc2af9b0e1
+        echo 'loading ACRN...'
+        multiboot2 /boot/acrn.bin  root=PARTUUID="e515916d-aac4-4439-aaa0-33231a9f4d83"
+        module2 /boot/bzImage Linux_bzImage
+
+      }
+
+   .. note::
+
+      Adjust this to your uuid and PARTUUID for the root= parameter with
```

Comment:
@fuzhongl, this line does not make sense. What does "lbkid cmdline" refer to? Please update it so your audience can understand it.

Reply:
Should be blkid; this command will show block UUID and PARTUUID info.